.eslintrc

```diff
@@ -7,11 +7,17 @@
     "angular",
     "html"
   ],
+  "overrides": [{
```

Comment:
Why do we need to disable this rule for Spec files now? Can you please add a comment explaining the rationale?

Reply:
That's because, with this rule, we need to have all the $Inject functions in the beginning be 

```
    beforeEach(inject(['XXX', function(XXX) {
      ...
```

instead of

```
    beforeEach(inject(function(XXX) {
```

However, the reason we use a list of imports in general is to guard against minification issues (since when the code in each snippet is minified, XXX is replaced with something else by the minification). I realized halfway through modifying the Spec files that minification is actually irrelevant for test files, so there is no need to do this for the test files, hence the exception.

I'll add a comment in the code to clarify.

Reply:
Thanks for the explanation!